package.json

```
@@ -7,74 +7,74 @@
     "@bytemd/plugin-math": "1.21.0",
     "@bytemd/plugin-mermaid": "1.21.0",
     "@bytemd/react": "1.21.0",
-    "@primer/octicons-react": "19.8.0",
-    "@primer/react": "36.7.1",
-    "@react-email/components": "0.0.14",
-    "@react-email/render": "0.0.12",
-    "@resvg/resvg-js": "2.6.0",
-    "@upstash/ratelimit": "1.0.1",
-    "@upstash/redis": "1.28.3",
-    "@vercel/analytics": "1.1.3",
+    "@primer/octicons-react": "19.10.0",
+    "@primer/react": "36.22.0",
+    "@react-email/components": "0.0.19",
+    "@react-email/render": "0.0.15",
+    "@resvg/resvg-js": "2.6.2",
+    "@upstash/ratelimit": "1.2.1",
+    "@upstash/redis": "1.31.6",
+    "@vercel/analytics": "1.3.1",
     "async-retry": "1.3.3",
     "bcryptjs": "2.4.3",
     "cookie": "0.6.0",
-    "date-fns": "3.3.1",
+    "date-fns": "3.6.0",
     "feed": "4.2.2",
     "is-in-subnet": "4.0.1",
-    "joi": "17.12.1",
-    "next": "14.1.0",
+    "joi": "17.13.3",
+    "next": "14.2.4",
     "next-connect": "0.13.0",
     "next-swr": "0.2.0-canary.0",
-    "node-pg-migrate": "6.2.2",
-    "nodemailer": "6.9.9",
+    "node-pg-migrate": "7.5.1",
+    "nodemailer": "6.9.14",
     "nprogress": "0.2.0",
     "parse-link-header": "2.0.0",
-    "pg": "8.11.3",
-    "pino": "8.18.0",
-    "react": "18.2.0",
+    "pg": "8.12.0",
+    "pino": "9.2.0",
+    "react": "18.3.1",
     "react-confetti": "6.1.0",
-    "react-dom": "18.2.0",
-    "react-icons": "5.0.1",
+    "react-dom": "18.3.1",
+    "react-icons": "5.2.1",
     "react-rewards": "2.0.4",
-    "recharts": "2.12.0",
+    "recharts": "2.12.7",
     "rehype-external-links": "3.0.0",
     "rehype-slug": "6.0.0",
     "satori": "0.10.13",
-    "slug": "8.2.3",
+    "slug": "9.1.0",
     "snakeize": "0.1.0",
     "styled-components": "5.3.11",
-    "swr": "2.2.4",
+    "swr": "2.2.5",
     "vis-network": "9.1.9"
   },
   "devDependencies": {
-    "@commitlint/cli": "18.6.1",
-    "@commitlint/config-conventional": "18.6.2",
+    "@commitlint/cli": "19.3.0",
+    "@commitlint/config-conventional": "19.2.2",
     "@faker-js/faker": "8.4.1",
-    "@testing-library/react": "15.0.0",
-    "@vitejs/plugin-react": "4.2.1",
-    "autoprefixer": "10.4.17",
+    "@testing-library/react": "16.0.0",
```

Na versão 16 a [breaking change](https://github.com/testing-library/react-testing-library/releases) é que precisa adicionar a dependência `@testing-library/dom` (agora, ela é uma peer-dependency da biblioteca).

```suggestion
    "@testing-library/dom": "10.2.0",
    "@testing-library/react": "16.0.0",
```

---

Mas nós não estamos utilizando a `@testing-library/dom`, ou estamos?

Não é melhor adicionar apenas quando precisar?

---

Nós realmente não usamos. Pelo o que vi da base de código da `@testing-library/react`, o [`render`](https://github.com/testing-library/react-testing-library/blob/a4744fa904bf11812c92093226c3805450472636/src/pure.js#L199) não utiliza a `@testing-library/dom`. Podemos adicionar quando formos criar novos testes que precisem disso.

O único incômodo seria caso houvesse necessidade de usar o [`debug`](https://github.com/testing-library/react-testing-library/blob/a4744fa904bf11812c92093226c3805450472636/src/pure.js#L162) (que utiliza o [`prettyDOM`](https://github.com/testing-library/react-testing-library/blob/a4744fa904bf11812c92093226c3805450472636/src/pure.js#L6)), e apenas ele, sem outra função que dependesse da `@testing-library/dom`. Então, precisaríamos adicioná-la como dependência para criar o teste, e depois remover (visto que o `debug` não irá para o `commit`).

---

Como é praticamente certo que logo iremos precisar, e tem essa questão do `debug`, então já adicionei. 👍